snappy/security.go

```diff
@@ -762,7 +768,12 @@ func generatePolicy(m *snapYaml, baseDir string) error {
 		if err != nil {
 			return err
 		}
+
+		// if no skill is specified, use the defaultSecurityPolicy
```

Comment:
// TODO This is not actually right. Even if there are skills, we still want to give the snap
// a default set of allowances, such as being able to read and write in its own directory
// and perhaps network access (we're still deciding on that one). So the real logic we
// want here is: give the snap a default set of permissions, and then whatever else the
// skills permit (migration or not). This is coming soon.